extensions/amp-analytics/amp-analytics.md

```diff
@@ -775,7 +779,34 @@ To configure a timer which times user events use:
 
 See the spec on [triggers](#triggers) for details on creating nested timer
 triggers. Note that using a timer trigger to start or stop a timer is not
-allowed.
+allowed. Please refer to the examples below on how to configure triggers based on a documents `hidden` and `visible` events.
+
+```json
+"triggers": {
+  "stopOnHiddenTimer": {
+    "on": "timer",
+    "timerSpec": {
+      "interval": 5,
+      "stopSpec": {
+        "on": "hidden",
+        "selector": ":root"
+      }
+    },
+    "request": "timerRequest"
+  },
+  "stopOnVisibleTimer": {
```

Comment:
I think a stop on hidden is sufficient. stop on visible is probably valid if you have a start on hidden, otherwise I find this use case not so useful. 

worth adding a start on visible which can be used to restart a timer once the doc turns visible again.